Description: Bump termcolor
 The code turns out to need version 1.4, even though upstream doesn't specify
 this as a dependency.
--- rust-ripgrep-14.0.3.orig/Cargo.toml
+++ rust-ripgrep-14.0.3/Cargo.toml
@@ -163,7 +163,7 @@ version = "0.4.5"
 version = "1.0.23"
 
 [dependencies.termcolor]
-version = "1.1.0"
+version = "1.4.0"
 
 [dependencies.textwrap]
 version = "0.16.0"
